exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -411,6 +404,7 @@ private ServerConnector createHttpsConnector(int port) throws Exception {
 
     // SSL Connector
     final ServerConnector sslConnector = new ServerConnector(embeddedJetty,
+        null, null, null, -1, -1,
```

Comment:
Why not use the configured values for HTTPS?

Reply:
good catch 👍